lesson07/assignment/sp1/src/components/Navbar.vue

```diff
@@ -0,0 +1,159 @@
+<template>
+  <section>
+    <!-- NAVBAR -->
+    <nav class="nav">
+      <ul class="nav nav-pills">
+        <li class="nav-item"><a id="homeIcon" href="#" data-toggle="tooltip" data-placement="bottom" title="Home">index</a></li>
+        <li class="nav-item">
+          <!-- use router-link for link to other views -->
+          <router-link to="/" class="nav-link">Landing</router-link>
+        </li>
+        <!-- "to" value is not a file path. it's a route. see index.js-->
+        <li class="nav-item">
+          <router-link to="/map" class="nav-link">Map</router-link>
+        </li>
+        <li class="nav-item">
+          <router-link to="/about" class="nav-link">About</router-link>
+        </li>
+        <li class="nav-item">
+          <a class="nav-link" href="#">social</a>
+        </li>
+        <li class="nav-item">
+          <button class="btn btn-info nav-link" type="button" data-toggle="collapse" data-target="#collapseExample" aria-expanded="false" aria-controls="collapseExample" data-placement="bottom" title="Click to Show/Hide">
+            Spread the word!
+          </button>
+        </li>
+      </ul>
+    </nav>
+
+
+    <!-- JUMOBTRON 1 CTA generic twitter button-->
+    <div class="collapse" id="collapseExample">
+        <div class="card card-body container jumbotron jumbotron-fluid CTAcontainer">
+          <div class="container">
+            <h1 class="display-4">Your newest little lifehack.</h1>
+            <span class="lead">Spread the love.</span>
+            <div class="CTAcontainer">
+              <p><i class="fab fa-twitter fa-5x twitterCTA"></i></p>
+              <a href="https://twitter.com/intent/tweet?button_hashtag=SeattlePGs&ref_src=twsrc%5Etfw" class="twitterIcon twitter-hashtag-button button" data-size="large" data-text="Hi! I just discovered Seattle&#39;s best resource for local playgrounds!"
+                data-related="" data-show-count="false" target="_blank">Tweet #SeattlePGs</a>
+            </div>
+
+            <p class='twitterIcon'>Hey there smartypants! Want to help this project? Go on and tweet about it! It's a win-win! ;D</p>
+          </div>
+        </div>
+    </div>
```

Comment:
The collapsed jumbotron hiding in the nav is a very cool user experience! Nicely done.